tensorflow/core/framework/bfloat16.cc

```diff
@@ -31,6 +32,18 @@ void FloatToBFloat16(const float* src, bfloat16* dst, int64 size) {
 #endif
 }
 
+void FloatToBFloat16(const Eigen::ThreadPoolDevice& d, const float* src,
+                     bfloat16* dst, int64 size) {
+  auto ParallelFloatToBFloat16 = [&](Eigen::Index start, Eigen::Index end) {
```

Comment:
could you please use `parallel_float_to_bfloat16` for lamdas to be consistent with a style guide. + the same for other lambdas.